addons/Clock/src/presenter.js

```diff
@@ -1417,18 +1422,21 @@ function AddonClock_create() {
 
         presenter.setAttr('h-hand', h);
         presenter.setAttr('m-hand', m);
-
-
     };
 
     presenter.onEventReceived = function (eventName) {
+        switch (eventName) {
+            case 'GradualShowAnswers':
+                presenter.isGradualShowAnswersActive = true;

```

Comment:
Obsługując event powinieneś sprawdzić czy funkcja GSA na pewno poszukuje konkretnie tego addonu w danej chwili. Inaczej, jeśli jest jakiś addon który GSA powinno obsłużyć przed Clock, to te dwa addony pokażą swoją odpowiedź jednocześnie, co się mija z założeniem GSA. Spójrz na ten przykład: 
https://test-8083-v1-dot-mauthor-dev.ew.r.appspot.com/embed/4841954720350208

Tutaj masz kod z innego addonu który działa: 
`presenter.onEventReceived = function (eventName, data) {

        if (!presenter.isModelValid) return;

        if (eventName === "ShowAnswers") {

            presenter.showAnswers();

        } else if (eventName === "HideAnswers") {

            presenter.hideAnswers();

        } else if (eventName === "GradualShowAnswers") {

            if (!presenter.isGradualShowAnswersActive) {

                presenter.isGradualShowAnswersActive = true;

            }

            if (data.moduleID === presenter.addonID) {

                presenter.gradualShowAnswers(parseInt(data.item, 10));

            }

        } else if (eventName === "GradualHideAnswers") {

            presenter.isGradualShowAnswersActive = false;

            presenter.hideAnswers();

        }

    };`